docker/create_admin_and_community.rb

```diff
@@ -0,0 +1,12 @@
+# 1. Create the community
+community_name = ENV['COMMUNITY_NAME'] || 'Dinosaur Community'
+Community.create(name: community_name, host: '0.0.0.0:3000')
+
+# 2. Create the admin user, ensure doesn't require confirmation
+username = ENV['COMMUNITY_ADMIN_USERNAME'] || 'admin'
+password = ENV['COMMUNITY_ADMIN_PASSWORD'] || 'password'
+email = ENV['COMMUNITY_ADMIN_EMAIL'] || 'codadict@noreply.com'
+
+User.create(username: username, password: password, email: email, is_global_admin: true, is_global_moderator: true, staff: true)
```

Comment:
Is there a reason that we can't just set `confirmed_at` here too?

Reply:
I tried this approach - it didn't work. I'm not sure why, to be honest. But re-thinking it, in the same way that it's required by design, we should also require the user to intentionally issue the command to match that. I wrote the instruction clearly in the docs.